engine/apps/base/messaging.py

```diff
@@ -43,6 +43,9 @@ def unlink_user(self, user):
     def is_enabled_for_organization(organization):
         return True
 
+    def is_configured_for_organization(self, organization):
```

Comment:
is this expected to be overridden in classes that subclass `BaseMessagingBackend`?

Reply:
ahh nvm, looking at [the other PR](https://github.com/grafana/oncall-private/pull/2634/files#diff-5308bafce8207e7b8e5b7049a10a9748055e408407efcb5215cf42ecaf64271aR17-R18), I understand now 🙂